src/app/zap-templates/zcl/data-model/chip/replacable-monitoring-cluster.xml

```diff
@@ -0,0 +1,324 @@
+    </command>
+  </cluster>
+
+  <cluster>
+    <name>Ink Cartridge Monitoring</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <description>Attributes and commands for monitoring ink cartridges in a device</description>
+    <code>0x007B</code>
+    <define>INK_CARTRIDGE_MONITORING</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+
+    <!-- Attributes -->
+    <attribute side="server" code="0x0000" define="INK_CARTRIDGE_MONITORING" type="percent" min="0" max="100" writable="false" isNullable="false">Condition</attribute>
+    <attribute side="server" code="0x0001" define="INK_CARTRIDGE_MONITORING" type="DegradationDirectionEnum" min="0" max="1" writable="false" isNullable="false">DegradationDirection</attribute>
+    <attribute side="server" code="0x0002" define="INK_CARTRIDGE_MONITORING" type="ChangeIndicationEnum" min="0" max="2" writable="false" isNullable="false" default="0" optional="false">ChangeIndication</attribute>
+    <attribute side="server" code="0x0003" define="INK_CARTRIDGE_MONITORING" type="boolean" writable="false" isNullable="false" optional="true">InPlaceIndicator</attribute>
+
+    <!-- Commands -->
+    <command source="client" code="0x00" name="ResetCondition" optional="true">
+      <description>Reset the condition of the replaceable to the non degraded state</description>
+    </command>
+  </cluster>
+
+  <cluster>
+    <name>Toner Cartridge Monitoring</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <description>Attributes and commands for monitoring toner cartridges in a device</description>
+    <code>0x007C</code>
+    <define>TONER_CARTRIDGE_MONITORING</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+
+    <!-- Attributes -->
+    <attribute side="server" code="0x0000" define="TONER_CARTRIDGE_MONITORING" type="percent" min="0" max="100" writable="false" isNullable="false">Condition</attribute>
+    <attribute side="server" code="0x0001" define="TONER_CARTRIDGE_MONITORING" type="DegradationDirectionEnum" min="0" max="1" writable="false" isNullable="false">DegradationDirection</attribute>
+    <attribute side="server" code="0x0002" define="TONER_CARTRIDGE_MONITORING" type="ChangeIndicationEnum" min="0" max="2" writable="false" isNullable="false" default="0" optional="false">ChangeIndication</attribute>
+    <attribute side="server" code="0x0003" define="TONER_CARTRIDGE_MONITORING" type="boolean" writable="false" isNullable="false" optional="true">InPlaceIndicator</attribute>
+
+    <!-- Commands -->
+    <command source="client" code="0x00" name="ResetCondition" optional="true">
+      <description>Reset the condition of the replaceable to the non degraded state</description>
+    </command>
+  </cluster>
+
+  <!-- Feature Map -->
+  <bitmap name="ReplaceableMonitoringFeature" type="BITMAP32">
```

Comment:
The patter for other clusters seems to be "ClusterNameFeature", but here we are going to have clusters with other names that all have "ReplaceableMonitoringFeature" in them.  I wish we had just used bareword "Features" for the other clusters.... but maybe we should consider doing that here.  @ReneJosefsen @andy31415 thoughts?

Reply:
I would also advocate for just bare `Feature` 👍

Reply:
It is a very good point @bzbarsky-apple, maybe we should generally do this for alias clusters? (here the interpretation is alias clusters is a single spec doc). I'm +1 for "Features".

Reply:
OK, let's just call this "Features"!

Reply:
Done